pkg/terra/client/gas_price_estimator_test.go

```diff
@@ -99,7 +102,7 @@ func TestGasPriceEstimators(t *testing.T) {
 			"uluna": sdk.NewDecCoinFromDec("uluna", sdk.MustNewDecFromStr("10")),
 		})
 		gpe := NewMustGasPriceEstimator([]GasPricesEstimator{cachingFCD, gpeFixed}, lggr)
-		lggr.On("Warnf", mock.Anything, mock.Anything, mock.Anything).Twice()
+		t.Cleanup(assertLogsLen(t, 1))
```

Comment:
This one was incorrect, but these weren't actually being checked before because the `lggr.AssertExpectations(t)` at the end was missing.